### ISSUE TITLE
docker-compose fails

### ISSUE THREAD

**Issue by gucchy55** [adamzammit/limesurvey-docker#41]
When I cloned the repository (tag: 6.1.4) and just ran "docker-compose up -d", it failed with the following:

```
Step 7/16 : RUN set -x;     curl -SL "$DOWNLOAD_URL" -o /tmp/lime.zip;     echo "$DOWNLOAD_SHA256 /tmp/lime.zip" | sha256sum -c - || exit 1;     unzip /tmp/lime.zip -d /tmp;     mv /tmp/lime*/* /var/www/html/;     mv /tmp/lime*/.[a-zA-Z]* /var/www/html/;     rm /tmp/lime.zip;     rmdir /tmp/lime*;     chown -R www-data:www-data /var/www/html;     mkdir -p /var/lime/application/config;     mkdir -p /var/lime/upload;     mkdir -p /var/lime/plugins;     mkdir -p /var/lime/sessions;     chown -R www-data:www-data /var/lime/sessions;     cp -dpR /var/www/html/application/config/* /var/lime/application/config;     cp -dpR /var/www/html/upload/* /var/lime/upload;     cp -dpR /var/www/html/plugins/* /var/lime/plugins
 ---> Running in 57696ec6de2f
+ curl -SL https://download.limesurvey.org/latest-master/limesurvey6.1.4+230620.zip -o /tmp/lime.zip
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   162  100   162    0     0     27      0  0:00:06  0:00:05  0:00:01    35
+ echo 64ddd7a7b0c854295882989e51564ca042f6b43ccfe164342688bf3f691e39d7 /tmp/lime.zip
+ sha256sum -c -
/tmp/lime.zip: FAILED
sha256sum: WARNING: 1 computed checksum did NOT match
+ exit 1
```

What is the cause of this problem?

**Comment by gucchy55** [comment]
I found that https://download.limesurvey.org/latest-master/limesurvey6.1.4+230620.zip is no longer available (only for 6.1.5?).
Should the repository be updated to 6.1.5?

**Comment by adamzammit** [comment]
You are correct - manually building will fail when the download file is no longer available. I've updated it now.
